ts/Stock/Navigator/Navigator.ts

```diff
@@ -1436,6 +1521,20 @@ class Navigator {
         // Add redraw events
         navigator.addChartEvents();
     }
+    /**
+     * Set the opposite property on navigator
+     *
+     * @private
+     */
+    public setNavigatorOpposite(): void {
```

Comment:
In prototyped methods, you don't have to add classnames. It looks strange: `navigator.setNavigatorOpposite()`, while `navigator.setOpposite()` is self explanatory